components/shared/side-navbar/NavSection.tsx

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable tailwindcss/no-custom-classname */
+import React from 'react';
+
+const NavSection = ({children, title}: {children: React.ReactNode, title: string}) => {
```

Comment:
Since two of the sections in the Navbar shared a similar pattern, I extracted this into a reusable component